packages/form-config-antd/src/components/DraftFileList/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -92,11 +92,6 @@ exports[`components/Antd/DraftFileList renders correctly: table 1`] = `
         "sorter": [Function],
         "title": "Module",
       },
-      Object {
-        "key": "_actions",
-        "render": [Function],
-        "title": "Actions",
-      },
```

Comment:
why has the table column change affected the form config package?

Reply:
the action column prop should not be merged with column prop this is the fix and they both should be kept separately instead of adding up each render